addons/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/blob/s3/S3BlobStoreConfiguration.java

```diff
@@ -204,8 +207,6 @@
      */
     public static final String DISABLE_PROXY_PROPERTY = "nuxeo.s3.proxy.disabled";
 
-    public static final ObjectLockRetentionMode DEFAULT_RETENTION_MODE = ObjectLockRetentionMode.GOVERNANCE;
```

Comment:
Deprecate just in case, as we're in 10.10?

Reply:
We agreed with @efge to remove these public constants as they were unlikely used by anyone